spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/channel/PostgresChannelMessageTableSubscriber.java

```diff
@@ -278,6 +285,19 @@ private static String getKey(Object input) {
 		return input == null ? null : UUIDConverter.getUUID(input).toString();
 	}
 
+	/**
+	 * Set the timeout for the notification polling.
+	 * If for the specified duration no notificiation are received the underlying connection is closed and re-established.
+	 * Setting a value of {@code Duration.ZERO} will disable the timeout and wait forever.
+	 * This might cause problems in DB failover scenarios.
+	 *
```

Comment:
No blank lines in the method Javadocs.
Please, move this new setter to the setters sections, which usually is after constructor.
Also add your name to the `@author` list.